reports/meta.tex

```diff
@@ -167,7 +172,9 @@ \subsubsection{ Mutual recursion cycles}
       )
     )
 \end{lstlisting}
-for which $\op(x) = \op_1(x)$ for all $x$, and $\op$ terminates iff $\op_1$ terminates.
+for which $\op(x) = \op_1(x)$ for all $x$, and $\op$ terminates iff $\op_1$ terminates
+\tth{Should we rename the funcdtion as \texttt{R(\_)} is already used on Pages 9 and 10?}
```

Comment:
The point of this section is to show that we can define an operator to which the rewriting rules of the previous section could apply. The previous section does not define "R" but rather express the shape assumptions on it.